assets/html/sign.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <link rel = "stylesheet" href = "../css/sign.css">
+    <!-- <meta name="viewport" content="width=device-width,initial-scale=1">-->
+    <title> WRITE YOUR WISH TO MAKE IT HAPPEN </title>
+  </head>
+  <body>
+    <div id="main-container" class="container">
+      <h4>sign in</h4>
+      <div class="signin-form">
+        <form action="" method="" id="user-signin" accept-charset="UTF-8">
+          <input class="form-control" type="text" id="name" name="user_name" placeholder="username" required>
+          <input class="form-control" type="password" id="password" name="password" placeholder="password" required>
+        </form>
+      </div>
+    </div>
+    <footer>
+      <div class="container">
+        <div class="row">
+          <div class="col">
+            <p id="app-title">
+              WRITE YOUR WISH
+              <br>
+              　TO MAKE IT HAPPEN
+            </p>
+          </div>
+          <div class="col">
+            <div id=credit>
```

Comment:
This has an id of credit, but it's not in quotation marks.  I'm not sure if this is getting the correct CSS

Reply:
thanks! will fix!

Reply:
ok done! new file will be pushed later